src/modules/taskAdmin.js

```diff
@@ -0,0 +1,25 @@
+export function addTask(description, tasks) {
+  const newIndex = tasks.length;
+  const newTask = {
+    description,
+    completed: false,
+    index: newIndex,
+  };
+  tasks.push(newTask);
+}
+
+export function deleteTask(taskIndex, tasks) {
+  console.log('Trying to delete task with index:', taskIndex);
+  const index = tasks.findIndex((task) => task.index === taskIndex);
+  console.log('Task index in tasks array:', index);
+  if (index !== -1) {
+    tasks.splice(index, 1);
+  }
+}
```

Comment:
[OPTIONAL] Using filter might have been better in this case. Instead of searching then using conditionals, filter takes care of all that for you